src/BannerAd.ts

```diff
@@ -0,0 +1,175 @@
+}
+
+const destroyAd = (adUnitId: string): void => {
+    AppLovinMAX.destroyBanner(adUnitId);
+}
+
+const showAd = (adUnitId: string): void => {
+    AppLovinMAX.showBanner(adUnitId);
+}
+
+const hideAd = (adUnitId: string): void => {
+    AppLovinMAX.hideBanner(adUnitId);
+}
+
+const setPlacement = (adUnitId: string, placement: string | null): void => {
+    AppLovinMAX.setBannerPlacement(adUnitId, placement);
+}
+
+const setCustomData = (adUnitId: string, customData: string | null): void => {
+    AppLovinMAX.setBannerCustomData(adUnitId, customData);
+}
+
+const updatePosition = (adUnitId: string, bannerPosition: AdViewPosition): void => {
+    AppLovinMAX.updateBannerPosition(adUnitId, bannerPosition);
+}
+
+const setExtraParameter = (adUnitId: string, key: string, value: any): void => {
+    AppLovinMAX.setBannerExtraParameter(adUnitId, key, value);
+}
+
+const setLocalExtraParameter = (adUnitId: string, key: string, value: any): void => {
+    AppLovinMAX.setBannerLocalExtraParameter(adUnitId, { [key]: value });
+}
+
+const startAutoRefresh = (adUnitId: string): void => {
+    AppLovinMAX.startBannerAutoRefresh(adUnitId);
+}
+
+const stopAutoRefresh = (adUnitId: string): void => {
+    AppLovinMAX.stopBannerAutoRefresh(adUnitId);
+}
+
+const addAdLoadedEventListener = (listener: (adInfo: AdInfo) => void) => {
+    addEventListener(ON_BANNER_AD_LOADED_EVENT, (adInfo: AdInfo) => listener(adInfo));
+}
+
+const removeAdLoadedEventListener = () => {
+    removeEventListener(ON_BANNER_AD_LOADED_EVENT);
+}
+
+const addAdLoadFailedEventListener = (listener: (errorInfo: AdLoadFailedInfo) => void) => {
+    addEventListener(ON_BANNER_AD_LOAD_FAILED_EVENT, (errorInfo: AdLoadFailedInfo) => listener(errorInfo));
+}
+
+const removeAdLoadFailedEventListener = () => {
+    removeEventListener(ON_BANNER_AD_LOAD_FAILED_EVENT);
+}
+
+const addAdClickedEventListener = (listener: (adInfo: AdInfo) => void) => {
+    addEventListener(ON_BANNER_AD_CLICKED_EVENT, (adInfo: AdInfo) => listener(adInfo));
+}
+
+const removeAdClickedEventListener = () => {
+    removeEventListener(ON_BANNER_AD_CLICKED_EVENT);
+}
+
+const addAdCollapsedEventListener = (listener: (adInfo: AdInfo) => void) => {
+    addEventListener(ON_BANNER_AD_COLLAPSED_EVENT, (adInfo: AdInfo) => listener(adInfo));
+}
+
+const removeAdCollapsedEventListener = () => {
+    removeEventListener(ON_BANNER_AD_COLLAPSED_EVENT);
+}
+
+const addAdExpandedEventListener = (listener: (adInfo: AdInfo) => void) => {
+    addEventListener(ON_BANNER_AD_EXPANDED_EVENT, (adInfo: AdInfo) => listener(adInfo));
+}
+
+const removeAdExpandedEventListener = () => {
+    removeEventListener(ON_BANNER_AD_EXPANDED_EVENT);
+}
+
+const addAdRevenuePaidListener = (listener: (adInfo: AdRevenueInfo) => void) => {
+    addEventListener(ON_BANNER_AD_REVENUE_PAID, (adInfo: AdRevenueInfo) => listener(adInfo));
+}
+
+const removeAdRevenuePaidListener = () => {
+    removeEventListener(ON_BANNER_AD_REVENUE_PAID);
+}
+
+// banner specific
```

Comment:
```suggestion
// Banner specific APIs
```

Reply:
Also, we should probably work with David after you merge this, on documenting all the public APIs like we do in our native SDKs / Unity Plugin.

Reply:
I'll add more comments and file a ticket for the API documentation.